tests/console/yast2_apparmor.pm

```diff
@@ -38,123 +40,98 @@ sub run {
     }
     assert_screen 'yast2_apparmor_enabled';
     # part 1: open profile mode configuration and check toggle/show all profiles
-    send_key 'alt-n';
+    wait_screen_change { send_key 'alt-n' };
```

Comment:
you should not need a wait_screen_change before an assert_screen because that offers a synchronisation point already. No point in waiting twice